blockdag/dag_test.go

```diff
@@ -625,13 +636,19 @@ func TestAcceptingInInit(t *testing.T) {
 	testNode.status = statusDataStored
 
 	// Manually add the test block to the database
-	err = db.Update(func(dbTx database.Tx) error {
-		err := dbStoreBlock(dbTx, testBlock)
-		if err != nil {
-			return err
-		}
-		return dbStoreBlockNode(dbTx, testNode)
-	})
+	testBlockBytes, err := testBlock.Bytes()
```

Comment:
This code is very similar to the code you wrote in accept.go - consider extracting to a function